### PR TITLE
Remove double-spacing

### PR DESCRIPTION
**Pull request by hestonhoffman**
Sorry for the huge PR 😅
Double-spacing in our partials is causing some issues with the Translation PRs, cos the translators copy the partials across and it causes the linter to fail. 

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Review by 0xnm — APPROVED** [review]
ok for RUM mobile

File: content/en/monitors/guide/suppress-alert-with-downtimes.md [hunk 1]
```diff
@@ -20,9 +20,13 @@ Use the [Downtime API][1] to manage advanced maintenance schedules format, or to
 
 This guide describes how to configure downtimes for the following use cases:
 
-* [Downtime over the weekend](#downtime-over-the-weekend)
-* [Downtime outside of business hours](#downtime-outside-of-business-hours)
-* [Recurring downtime on nth weekday of the month](#recurring-downtime-on-the-nth-weekday-of-the-month)
```
Comment (drichards-87) [inline content/en/monitors/guide/suppress-alert-with-downtimes.md:25]: Just confirming that it was intentional to add new content here (versus only adjusting the double-spacing).

**Review by drichards-87 — APPROVED** [review]
Left you a question, but otherwise everything looks good. PR is approved.

File: content/en/monitors/guide/suppress-alert-with-downtimes.md (hunk 1 above)
Comment (hestonhoffman) [reply content/en/monitors/guide/suppress-alert-with-downtimes.md]: Good call, I'm reverting the change to this file

**Pull request merged**